src/apps/logging/logging.cpp

```diff
@@ -65,7 +65,7 @@ namespace ccfapp
   enum class LoggerErrors : jsonrpc::ErrorBaseType
   {
     UNKNOWN_ID =
```

Comment:
As discussed with @eddyashton, app-defined error codes are not working properly since the frontend refactoring. Without this change, a missing log entry when `LOG_get` would return the wrong error (`MESSAGE_EMPTY`). This is a temporary fix to return the generic `"message"` error code instead. I'll ticket that.